src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -245,6 +247,11 @@ public function getValueFromFieldDescription(
         FieldDescriptionInterface $fieldDescription,
         array $params = []
     ) {
+        if ([] !== $params) {
+            @trigger_error('Passing a third parameter is deprecated since sonata-project/admin-bundle 3.x and will be ignored in 4.0.', E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(sprintf(
                'Passing argument 3 to %s() is deprecated since sonata-project/admin-bundle 3.x'
                .' and will be ignored in version 4.0.',
                __METHOD__
                ), E_USER_DEPRECATED);
```